.github/workflows/docsbuild-release.yaml

```
@@ -0,0 +1,73 @@
+name: ansible-release-porting-guide
+on:
+  workflow_dispatch:
+    inputs:
+      ansible_version:
+        description: 'Release Version. Example : 11.1.0'
+        required: true
+      ansible_major_version:
+        description: 'Exmaple 11'
+        required: true
+
+env:
+  CI_COMMIT_MESSAGE: 'Add Ansible community "${{ inputs.ansible_version }}" porting guide'
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  CI_PR_BODY: |
+
+      ##### SUMMARY
+
+      ##### ISSUE TYPE
+
+      -   Docs Pull Request
```

Are you sure markdown would parse the multispace correctly?
```suggestion
      - Docs Pull Request
```